### PR TITLE
Add documentation for `sqlalchemy_session_persistence`

### PR DESCRIPTION
**Pull request by QuantumGhost**

### CODE REVIEW

File: docs/orms.rst [hunk 1]
```diff
@@ -322,6 +322,22 @@ To work, this class needs an `SQLAlchemy`_ session object affected to the :attr:
     .. attribute:: force_flush
 
         Force a session flush() at the end of :func:`~factory.alchemy.SQLAlchemyModelFactory._create()`.
+        
+        This option is deprecated. Use ``sqlalchemy_session_persistence`` instead.
```
Comment (jeffwidman) [inline docs/orms.rst]: Can you style this with `.. note::` so that it stands out that it's deprecated... ?

File: docs/orms.rst [hunk 2]
```diff
@@ -322,6 +322,22 @@ To work, this class needs an `SQLAlchemy`_ session object affected to the :attr:
...
+    
+    .. attribute:: sqlalchemy_session_persistence
```
Comment (jeffwidman) [inline docs/orms.rst]: Can you move this attribute above the `force_flush` attribute? I know it's not alphabetical, but I want someone reading the docs to see this before the deprecated one.

**Comment by QuantumGhost** [comment]
Ok, I'll fix these two issues.

**Comment by QuantumGhost** [comment]
Updated.

**Comment by jeffwidman** [comment]
Thank you!

**Pull request merged**